### PR TITLE
fix(UI): Resolved discrepancies in solution dropdown text

### LINKED ISSUE #55096: Discrepancies in solution dropdown text on /setting and /certification
## Description

On /settings, the solution dropdown shows its items as "Front End" and "Back End":

<details>
  <summary>Image</summary>

Screenshot 2024-06-06 at 00 25 45
</details>

While on the /certification page, the items are "solution" and "source":

<details>
  <summary>Image</summary>

Screenshot 2024-06-06 at 00 25 57

</details>

## Affected page:

- /settings#cert-information-security-and-quality-assurance
- /certification/username/information-security-and-quality-assurance

## Changes needed

I'm not sure if the discrepancies are intended. If they are not, I think the items should be "solution" and "source", since the project page asks for those links:

<details>
  <summary>Image</summary>

Screenshot 2024-06-06 at 00 37 11
</details>

Additionally: 
- "solution" and "source" should be capitalized
- There should be a space between the text and the icon

### PR DESCRIPTION
**Pull request by Lalith246**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #55096

### CODE REVIEW

**Review by anandfresh — COMMENTED** [review]
LGTM

**Comment by a2937** [comment]
Hi there,

Thanks for creating this pull request.

The linked issue has not been triaged yet, and a solution has not been agreed upon. Once the issue is open for contribution, you are welcome to update this pull request to reflect the issue consensus. Until the issue is open for contribution, we will not be able to review your pull request.

**Review by huyenltnguyen — COMMENTED** [review]
The changes look good to me. There is just a small cleanup left.

File: client/src/components/solution-display-widget/index.tsx [hunk 1]
```diff
@@ -144,7 +144,7 @@ export function SolutionDisplayWidget({
             rel='noopener noreferrer'
             target='_blank'
           >
-            {t('buttons.frontend')}{' '}
```
[the remark this answers is not in the document]
Comment (Lalith246) [reply client/src/components/solution-display-widget/index.tsx:147]: Want me to make the changes and update the pull request?

[the remark this answers is not in the document]
Comment (Lalith246) [reply client/src/components/solution-display-widget/index.tsx:147]: Updated the PR.

**Review by huyenltnguyen — APPROVED** [review]

**Review by ahmaxed — APPROVED** [review]

**Pull request merged**